crates/meroctl/src/output.rs

```diff
@@ -1,12 +1,12 @@
 use clap::ValueEnum;
-use color_eyre::owo_colors::OwoColorize;
+use comfy_table::{Cell, Table};
 use serde::Serialize;
 
 #[derive(Clone, Copy, Debug, Default, ValueEnum)]
 pub enum Format {
     Json,
     #[default]
-    PlainText,
+    PrettyText,
```

Comment:
this will require a documentation update (`--output-format pretty-text`), since there's no other option to text than pretty text, I'd say we make this just `Text` or better yet, `Human`

cc: @fbozic

Reply:
Done.